charts/workbench-operator/templates/metrics-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "workbench-operator.fullname" . }}-controller-manager-metrics-service
+  labels:
+    control-plane: controller-manager
+  {{- include "workbench-operator.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.metricsService.type }}
+  selector:
+    control-plane: controller-manager
+  {{- include "workbench-operator.selectorLabels" . | nindent 4 }}
+  ports:
+	{{- .Values.metricsService.ports | toYaml | nindent 2 }}
```

Comment:
```suggestion
  {{- .Values.metricsService.ports | toYaml | nindent 2 }}
```

Reply:
it's generated code.

Reply:
https://github.com/arttor/helmify/pull/153